webapp/components/post_view/components/post_info.jsx

```diff
@@ -23,23 +23,26 @@ export default class PostInfo extends React.Component {
     constructor(props) {
         super(props);
 
-        this.handleDropdownClick = this.handleDropdownClick.bind(this);
+        this.handleDropdownOpened = this.handleDropdownOpened.bind(this);
         this.handlePermalink = this.handlePermalink.bind(this);
         this.removePost = this.removePost.bind(this);
         this.flagPost = this.flagPost.bind(this);
         this.unflagPost = this.unflagPost.bind(this);
     }
 
-    handleDropdownClick(e) {
-        var position = $('#post-list').height() - $(e.target).offset().top;
-        var dropdown = $(e.target).closest('.col__reply').find('.dropdown-menu');
+    handleDropdownOpened() {
+        this.props.handleDropdownOpened(true);
+
+        const position = $('#post-list').height() - $(this.refs.dropdownToggle).offset().top;
+        const dropdown = $(this.refs.dropdown);
+
         if (position < dropdown.height()) {
             dropdown.addClass('bottom');
         }
     }
 
     componentDidMount() {
-        $('#post_dropdown' + this.props.post.id).on('shown.bs.dropdown', () => this.props.handleDropdownOpened(true));
+        $('#post_dropdown' + this.props.post.id).on('shown.bs.dropdown', this.handleDropdownOpened);
```

Comment:
That would require us to change it to React-Bootstrap, but I don't think that's priority right now.

The other stuff with the height function is a bigger change since it needs to know stuff about the position of the parent post list and I didn't think it was worth it when this is only to fix this rare case.